styles/about.css

```diff
@@ -0,0 +1,118 @@
+html, body {
+    width: 100vw;
+    min-height: 100vh;
+    margin: 0;
+    padding: 0;
+}
+body {
+    display: grid;
+    grid-template-columns: 100vw;
+    grid-template-rows: 15vh auto 15vh;
+    justify-items: stretch;
+    align-items: stretch;
+    font-family: 'Courier New', Courier, monospace;
+    color: white;
+    background-color: black;
+}
+
+header {
+    background-color: black;
+    grid-column: 1/2;
+    grid-row: 1/2;
+}
+
+li h1 {
+    font-size: 2em;
+    margin-left: 70px;
+    margin-right: 40px;
+}
+
+nav {
+    grid-row: 1/2;
+    grid-column: 1/2;
+    margin: 0;
+    padding: 20px;
+    align-self: center;
+}
+
+nav ul {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+}
+
+ul {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+    color: white;
+}
+
+a:hover {
+    font-family: "Courier New", Courier, monospace;
+    letter-spacing: 3px;
+    word-spacing: -5px;
+    color: white;
+    font-weight: 700;
+    text-decoration: none;
+    font-style: italic;
+    font-variant: normal;
+    text-transform: uppercase;
+}
+main {
+    display: flex;
+    justify-items: center;
+    align-items: center;
+    flex-direction: column;
+    grid-column: 1/2;
+    grid-row: 2/3;
+    color: black;
+    overflow: hidden;
+}
+
+main div img:first-of-type {
```

Comment:
Consider adding a class to the div that's grouping the photo and name image, as well as the images themselves to make it clearer what purpose they are serving in this layout. Selecting solely by the structure can be difficult to reason bout where it will apply. And the first and last of types would be more appropriate if there were a collection of otherwise similar items (like a list of things or paragraphs) and stylistically you want the beginning or end to behave differently (but still related). Here, the first item and last item as more like a photo and a label, so they aren't just two images, they have real roles to play.